docs/docs/Support/troubleshooting.mdx

```diff
@@ -193,10 +195,66 @@ To fully remove a Langflow Desktop macOS installation, you must also delete `~/.
     rm -rf .langflow
     ```
 
-## MCP server issues
+## Langflow MCP issues {#mcp}
 
-- [Use MCP Inspector to test and debug flows](/mcp-server#test-and-debug-flows)
-- [Troubleshooting MCP server](/mcp-server#troubleshooting-mcp-server)
+The following issues can occur when using Langflow as an MCP server or client.
+
+### Claude for Desktop not using MCP server tools correctly
+
+If Claude for Desktop isn't using your server's tools correctly, you may need to explicitly define the path to your local `uvx` or `npx` executable file in the `claude_desktop_config.json` configuration file.
+
+1. To find your UVX path, run `which uvx`.
+
+    To find your NPX path, run `which npx`.
+
+2. Copy the path, and then replace `PATH_TO_UVX` or `PATH_TO_NPX` in your `claude_desktop_config.json` file.
```

Comment:
```suggestion
2. In your `claude_desktop_config.json` file, add the path to your Langflow MCP server configuration, as shown in the following examples.
```

The original wording implies they would find that placeholder already in the config file, but I don't think that's the case?